### PR TITLE
feat: DAH-2587 translations not on listing object

### PR DESCRIPTION
**Pull request by tallulahkay**
## Description

Gets translations for fields that aren't directly on the listing object. *Note: Tests are failing because this PR includes changes from https://github.com/SFDigitalServices/sf-dahlia-web/pull/2255. Tests will be fixed in that PR!*

## Jira ticket

DAH-2587

### Version Control

- [ ] branch name begins with `angular` if it contains updates to Angular code
- [x] branch name contains the Jira ticket number
- [x] PR name follows `type: TICKET-NUMBER Description` format, e.g. `feat: DAH-123 New Feature`

### Code quality

- [x] [the set of changes is small](https://google.github.io/eng-practices/review/developer/small-cls.html#what-is-small)
- [x] [if the set of changes cannot be small, reviewers have been forewarned](https://google.github.io/eng-practices/review/developer/small-cls.html#cant)
- [x] code meets test coverage thresholds
- [x] code is properly formatted
- [x] code is linted
- [x] code irrelevant to the ticket is not modified e.g. changing indentation due to automated formatting
- [x] automated tests pass
- [x] if the code changes the UI, it matches the UI design exactly

### Review instructions

- [x] instructions specify which environment(s) it applies to
- [x] instructions have already been performed at least once
- [x] instructions can be followed by PA testers
- [x] instructions specify if it can only be followed by an engineer

### Request review

- [x] PR has `needs review` label
- [x] Use `Housing Eng` group to automatically assign reviewers, and/or assign specific engineers
- [ ] If time sensitive, notify engineers in Slack

### CODE REVIEW

File: app/services/force/listing_service.rb [hunk 1]
```diff
@@ -41,12 +41,11 @@ def self.listing(id, opts = {})
       else
         puts 'test is NOT enabled'
       end
-      
-      listing_translations = @cache.read("/ListingDetails/#{id}/translations")
-      # TODO: DAH-2451
-      # @cache.fetch("/ListingDetails/#{id}/translations") do
-      #   # call translation service
-      # end
+
```
Comment (tallulahkay) [inline app/services/force/listing_service.rb:44]: This change is from PR https://github.com/SFDigitalServices/sf-dahlia-web/pull/2255

**Review by cade-exygy — APPROVED** [review]
Looks Good!

File: app/javascript/modules/listings/SharedHelpers.tsx [hunk 2]
```diff
@@ -230,12 +235,18 @@ export const getImageCardProps = (
   }
 }
 
-export const getEventNote = (listingEvent: ListingEvent) => {
+export const getEventNote = (
+  listingEvent: ListingEvent,
+  translations: RailsTranslations,
+  fieldName: string
+) => {
```
Comment (jimlin-sfgov) [inline app/javascript/modules/listings/SharedHelpers.tsx:242]: suggestion (non-blocking): Avoid modifying this function by passing translated text to this function, e.g.

```js
events={listing.Information_Sessions?.map((informationSession) => {
  return {
    dateString:
      informationSession.Date && localizedFormat(informationSession.Date, "LL"),
    timeString: getEventTimeString(informationSession),
    note: getEventNote(
      {
        ...informationSession,
        getTranslatedString(
          informationSession.Venue,
          listing.translations,
          `${informationSession.Id}.Information_Sessions.Venue__c`
        )
      },
    )
  }
})}
```

**Review by jimlin-sfgov — APPROVED** [review]
LG

one non-blocking suggestion

**Pull request merged**